src/base/orgs/View/Projects.svelte

```diff
@@ -1,32 +1,44 @@
 <script lang="ts">
+  import { navigate } from "svelte-routing";
+  import { onMount } from "svelte";
   import type { Config } from "@app/config";
-  import { Org } from "@app/base/orgs/Org";
-  import type * as proj from "@app/project";
+  import * as proj from "@app/project";
   import Loading from "@app/Loading.svelte";
   import Message from "@app/Message.svelte";
   import Widget from '@app/base/projects/Widget.svelte';
-  import Anchor from './Anchor.svelte';
-  import { formatCommit } from "@app/utils";
   import type { Profile } from "@app/profile";
+  import type { ProjectInfo, Anchor, PendingAnchor } from "@app/project";
+  import { Seed } from "@app/base/seeds/Seed";
+  import AnchorActions from "@app/base/profiles/AnchorActions.svelte";
 
   export let profile: Profile;
   export let config: Config;
   export let account: string | null;
 
-  const updateRecords = () => {
-    getProjects = queryProjects;
+  let anchors: Record<string, Anchor> = {};
+  let pendingAnchors: Record<string, PendingAnchor> = {};
+
+  const loadAnchors = async () => {
+    const [pending, confirmed] = await Promise.all([
+      profile.pendingAnchors(config),
+      profile.confirmedAnchors(config),
+    ]);
+
+    anchors = confirmed;
+    pendingAnchors = pending;
   };
 
-  $: queryProjects = async (): Promise<(proj.Project | proj.PendingProject)[]> => {
-    if (profile.org) {
-      if (account) {
-        const result = await profile.org.isMember(account, config);
-        return result ? profile.org.getAllProjects(config) : profile.org.getProjects(config);
-      }
-    }
-    return [];
+  const onClick = (project: ProjectInfo) => {
+    navigate(
+      proj.path({
+        urn: project.urn,
+        addressOrName: profile.name ?? profile.address,
+        revision: project.head,
```

Comment:
With this we go again to the `project.head` of a project instead of the latest anchor. Is this desired? Maybe we could navigate to the latest anchor of the project like `revision: anchors[project.urn].anchor.stateHash ?? project.head` or something similar.

Reply:
Yeah, I think latest anchor is actually going to be confusing for users. I'd rather they click on the anchor button to get to that state, but users will expect to see the latest code whether anchored or not, imo.

Reply:
Ok good for me 👍